go/src/meguca/db/sql/posts/increment_rcount.sql

```diff
@@ -0,0 +1,3 @@
+update main
```

Comment:
There will be no `id = 'rcount'` row in the DB, if the DB was generated freshly and not from a previous version upgrade. Probably need to init the row to 0 in `db/init.go:initDB()`. 

Now that I think about it, the same will happen with current roulette status, so mind initing that to 0 too?

Reply:
roulette gets initialized in init.sql.  I'll add rcout there too.

Reply:
Oh, did not notice. Good then.